theme_paptic/__manifest__.py

```diff
@@ -4,7 +4,7 @@
     'category': 'Theme/Corporate',
     'summary': 'Consultancy, Design, Tech, Computers, IT, Blogs',
     'sequence': 110,
-    'version': '2.0.0',
+    'version': '2.1.0',
```

Comment:
@xlu-odoo I am not sure what you did but you have a master commit of Brieuc in your branch :/ Did you fetch the branch I rebased like I warned you about and/or rebased it on 14.5 by yourself?